arkouda/categorical.py

```diff
@@ -200,7 +200,10 @@ def from_return_msg(cls, rep_msg) -> Categorical:
         eles = json.loads(rep_msg)
         codes = create_pdarray(eles["codes"])
         cats = Strings.from_return_msg(eles["categories"])
-        na_code = create_pdarray(eles["_akNAcode"])
+        if "_akNAcode" in eles:
+            na_code = create_pdarray(eles["_akNAcode"])
+        else:
+            na_code = None
```

Comment:
`ak.Categorical._akNAcode` should always be defined. Looking at the `init`, there are cases where it will not be a `pdarray` (which means we there are alterations we need to make unrelated to this PR). However, in this case, `_akNAcode` should always be defined and should always be a `pdarray`.

Reply:
OK I think I did something stupid there, working to get that fixed now, thanks for pointing out.